app/code/Magento/ReviewsStorefront/Model/CatalogRepository.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\ReviewsStorefront\Model;
+
+use Magento\ReviewsStorefront\Model\Storage\Client\CommandInterface;
+use Magento\ReviewsStorefront\Model\Storage\Client\DataDefinitionInterface;
+use Magento\ReviewsStorefront\Model\Storage\State;
+use Magento\Framework\Exception\BulkException;
+use Magento\Framework\Exception\CouldNotSaveException;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Repository for storing data to data storage.
+ */
+class CatalogRepository
```

Comment:
CatalogRepository is not used now, StorageRepository used instead. Eliminate this class